### ISSUE TITLE
Lara model problem in TR5

### ISSUE THREAD

**Issue by RICCIARDI-Adrien** [opentomb/OpenTomb#550]
Lara's holsters are missing in TR5 first level (and probably others) :
![capture d ecran_2018-06-09_11-32-05](https://user-images.githubusercontent.com/4292857/41190125-617f9b68-6bd9-11e8-8d80-9d4afa565b57.png)
![capture d ecran_2018-06-09_11-32-16](https://user-images.githubusercontent.com/4292857/41190126-66493d02-6bd9-11e8-99a9-bf4f56cc0814.png)

**Comment by TeslaRus** [comment]
yep, it caused by weapon system refactoring - I forgot why I made two flags for mesh replace and miss it...
